exporters/ostream/include/opentelemetry/exporters/ostream/metric_exporter.h

```diff
@@ -0,0 +1,66 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#ifndef ENABLE_METRICS_PREVIEW
+
+#  include <iostream>
+#  include <string>
+#  include "opentelemetry/common/spin_lock_mutex.h"
+#  include "opentelemetry/nostd/span.h"
+#  include "opentelemetry/sdk/metrics/instruments.h"
+#  include "opentelemetry/sdk/metrics/metric_exporter.h"
+#  include "opentelemetry/sdk/metrics/recordable.h"
+#  include "opentelemetry/version.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace metrics
+{
+
+/**
+ * The OStreamMetricExporter exports record data through an ostream
+ */
+class OStreamMetricExporter final : public opentelemetry::sdk::metrics::MetricExporter
+{
+public:
+  /**
+   * Create an OStreamMetricExporter. This constructor takes in a reference to an ostream that the
+   * export() function will send span data into.
+   * The default ostream is set to stdout
+   */
+  explicit OStreamMetricExporter(std::ostream &sout = std::cout) noexcept;
+
+  /**
+   * Export
+   * @param records a span of unique pointers to metrics recordables
+   */
+  sdk::common::ExportResult Export(
+      const nostd::span<std::unique_ptr<opentelemetry::sdk::metrics::Recordable>> &records) noexcept
```

Comment:
Can we modify this class, and the base class to accept span of [MetricData](https://github.com/open-telemetry/opentelemetry-cpp/blob/7485102bfaa68f8a5d4d70312c60cb928e70f00c/sdk/include/opentelemetry/sdk/metrics/data/metric_data.h#L24)

Reply:
Done